src/docs/automation-and-resiliency/openshift-project-resource-quotas.md

```diff
@@ -26,12 +27,13 @@ New project sets provisioned in **all clusters** of the BC Gov Private Cloud Paa
 
 **Note**: The Platform Services team may approach your team to discuss a downgrade in your quota if they find your application isn't using all of the resources within the current quota.
 
+If the default allocations aren't sufficient for your application, [you can ask for a quota increase](/request-quota-increase-for-openshift-project-set/). You'll need a Sysdig dashboard that shows that your application needs more of a specific resource type (CPU, RAM or storage) in a specific namespace. Provide this proof to the Platform Services team before they can approve a quota increase that you submit in the [Platform Project Registry](https://registry.developer.gov.bc.ca/public-landing).
+
 ## On this page
 - [CPU quotas](#cpu-quotas)
 - [Memory quotas](#memory-quotas)
 - [Storage quotas](#storage-quotas)
-
-If the default allocations aren't sufficient for your application, [you can ask for a quota increase](/request-quota-increase-for-openshift-project-set/). You'll need a Sysdig dashboard that shows that your application needs more of a specific resource type (CPU, RAM or storage) in a specific namespace. Provide this proof to the Platform Services team before they can approve a quota increase that you submit in the [Platform Project Registry](https://registry.developer.gov.bc.ca/public-landing).
```

Comment:
Did you intend to remove these lines @Pilargit12 ?

Reply:
I moved those above the On this page section instead